### ISSUE TITLE
[2.0b1]Bad pressure input on window focus change

### ISSUE THREAD

**Issue by pachuco** [callaa/Drawpile#364]
When switching to another window and pressing on DP canvas with stylus(thus regaining window focus), you get a large dot of max pressure, followed by normal pressure recognition.
I have tested for this bug in 1.0.4, it does not appear to manifest.

**Comment by Wade821** [comment]
Yeah, I'm not sure why this happens. Probably because of how it captures pen events or something.

**Comment by pachuco** [comment]
I don't see any such thing anymore in Drawpile 2.1.14.

**Comment by Wade821** [comment]
I do, I'm using Drawpile with Wintab mode (Not Windows Ink).

**Comment by Wade821** [comment]
> (I've taken over development on Drawpile, so I'm going through these tickets and organizing them.)
> 
> @Wade821 is that still happening?

Yep, in both versions of QT released for 2.2.
